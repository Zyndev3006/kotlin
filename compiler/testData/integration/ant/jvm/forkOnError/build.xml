<project name="Ant Task Test" default="build">
    <taskdef resource="org/jetbrains/kotlin/ant/antlib.xml" classpath="${kotlin.lib}/kotlin-ant.jar"/>
    <target name="build">
        <kotlinc src="${test.data}" output="${temp}/fork.jar" fork="true">
            <jvmarg value="-Xms64m"/>
            <jvmarg value="-Xmx128m"/>
            <jvmarg value="-Xmn32m"/>
            <jvmarg value="-XX:ReservedCodeCacheSize=64m"/>
            <jvmarg value="-XX:CICompilerCount=4"/>
            <jvmarg value="-XX:+PrintCommandLineFlags"/>

            <compilerarg line="-jvm-target 1.8"/>
            <compilerarg value="-Xno-call-assertions"/>
            <compilerarg value="-Xno-param-assertions"/>
<!--            <compilerarg value="-Xreport-perf"/>-->
            <compilerarg value="-progressive"/>
        </kotlinc>
    </target>
</project>
