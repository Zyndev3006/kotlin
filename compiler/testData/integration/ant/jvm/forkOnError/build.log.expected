OUT:
Buildfile: [TestData]/build.xml

build:
  [kotlinc] Compiling [[TestData]] => [[Temp]/fork.jar]
  [kotlinc] -XX:CICompilerCount=4 -XX:InitialHeapSize=67108864 -XX:MaxHeapSize=134217728 -XX:MaxNewSize=33554432 -XX:NewSize=33554432 -XX:+PrintCommandLineFlags -XX:ReservedCodeCacheSize=67108864 -XX:+UseCompressedClassPointers -XX:+UseCompressedOops -XX:-UseLargePagesIndividualAllocation -XX:+UseParallelGC
  [kotlinc] syntaxError.kt:4:5: error: unresolved reference: println1
  [kotlinc]     println1("hello fork")
  [kotlinc]     ^

ERR:

BUILD FAILED
[TestData]/build.xml:4: Compile failed; see the compiler error output for details.

Total time: [time]

Return code: 1
